developer-tools/scaleway-cli/quickstart.mdx

```diff
@@ -0,0 +1,146 @@
+
+</Message>
+
+## How to install the Scaleway CLI locally
+
+Download the Scaleway CLI using a package manager according to your operating System:
+
+### macOS
+
+Install the latest stable release on macOS using [Homebrew](https://formulae.brew.sh/formula/scw):
+
+```sh
+brew install scw
+```
+
+### Arch Linux
+
+Install the latest stable release on Arch Linux from [official repositories](https://archlinux.org/packages/community/x86_64/scaleway-cli/) using `pacman`:
+
+```sh
+pacman -S scaleway-cli
+```
+
+### Windows
+
+Install the latest stable release on Windows using [Chocolatey](https://chocolatey.org/packages/scaleway-cli):
+
+```powershell
+choco install scaleway-cli
+```
+
+<Message type="tip">
+Refer to the [Scaleway CLI repository](https://github.com/scaleway/scaleway-cli) for alternative installation methods.
+</Message>
+
+## How to configure the Scaleway CLI
+
+1. Open a terminal and type the following command to configure the Scaleway CLI on your machine:
+
+    ```
+    scw init
+    ```
+
+    <Message type="note">
+    If you have already configured the CLI, you will see a message like the following:
+
+    ```
+    Current config is located at /path/to/config.yaml
+    access_key: your-access-key
+    secret_key: your-secret-key
+    default_organization_id: ID-of-your-organization
+    default_project_id: ID-of-your-project
+    default_region: your-default-region (fr-par)
+    default_zone: your-default-zone (fr-par-1)
+
+    Do you want to override the current config? (Y/n):
+    ```
+    Type `n` if you do not want to edit your configuration.
+    </Message>
+
+2. The following message displays:
+
+    ```
+    Enter a valid secret-key or an email
+    ```
+
+    Enter either the email address for your account (you will then be asked for your password), or your [API Secret Key](/console/my-project/how-to/create-ssh-key) when prompted by the following message:
+
+3. The following message displays:
+
+    ```
+    To improve this tool we rely on diagnostic and usage data.
+    Sending such data is optional and can be disabled at any time by running "scw config set send-telemetry=false".
+    Do you want to send usage statistics and diagnostics? (Y/n):
+    ```
+
+    Type `y` or `n` according to your preference.
+
+4. The following message displays:
+
+    ```
+    To fully enjoy Scaleway CLI we recommend you install autocomplete support in your shell.
+    Do you want to install autocomplete? (Y/n):
+    ```
+
+    Type `y` or `n` according to your preference.
+
+5. The following message displays:
+
+    ```
+    To enable autocomplete, scw needs to update your shell configuration.
+    What type of shell are you using (default: zsh):
+    ```
+
+    Press enter if the default shell type displayed is the one you are using. Otherwise, enter the correct shell type you are using.
+
+6. The configuration is complete and you are prompted with the message below:
```

Comment:
Step should start with an action verb. I think this shouldn't be a step at all, since there is no action to take. Maybe just remove the number and leave it as extra info, continuing steps from 6 below.